repo/meta/schema/v3-repo-schema.json

```diff
@@ -65,6 +113,49 @@
           },
           "additionalProperties": false
         },
+        "cli": {
+            "additionalProperties": false,
+            "properties": {
+                "binaries": {
+                    "additionalProperties": false,
+                    "minProperties": 1,
+                    "properties": {
+                        "darwin": {
+                            "additionalProperties": false,
+                            "properties": {
+                                "x86-64": {
```

Comment:
if the name of this property changed to `x86_64` javascript will still be able to use dot-notation to address the field instead of having to use map notation:

`.resource.cli.binaries.darwin.x86_64` vs `.resource.cli.binaries.darwin['x86-64']`

Most C style languages allow `_` in variable/method/class/struct names where as `-` usually isn't allowed.